checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakAnnotatedTypeFactory.java

```diff
@@ -101,6 +115,15 @@ public void postAnalyze(ControlFlowGraph cfg) {
     MustCallConsistencyAnalyzer mustCallConsistencyAnalyzer =
         new MustCallConsistencyAnalyzer(this, this.analysis);
     mustCallConsistencyAnalyzer.analyze(cfg);
+
+    // Inferring owning annotations for final owning fields
+    if (getWholeProgramInference() != null && cfg != null) {
```

Comment:
When can `cfg` be null here? That seems like it might be a bug?

Reply:
I couldn't find an example in which cfg be null. I think I added this by mistake.